src/main/java/com/troubleshooting/model/EnvironmentsModel.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * Model object for a collection of container environment variables
+ */
+package com.troubleshooting.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class EnvironmentsModel {
+
+    List<EnvironmentModel> envList = new ArrayList<>();
+
+    public void addEnv(EnvironmentModel environmentModel){
```

Comment:
Consider adding a method that accepts a collection: addAll(Collection<EnvironmentModel>)